controllers/controlplane/nestedetcd_controller.go

```diff
@@ -32,14 +27,12 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/client-go/kubernetes/scheme"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	ctrlcli "sigs.k8s.io/controller-runtime/pkg/client"
 
-	controlplanev1alpha4 "sigs.k8s.io/cluster-api-provider-nested/apis/controlplane/v1alpha4"
+	capnapis "sigs.k8s.io/cluster-api-provider-nested/apis/controlplane/v1alpha4"
```

Comment:
nit. the convention is `clusterv1` eg - https://github.com/kubernetes-sigs/cluster-api/blob/master/controllers/cluster_controller_phases_test.go#L30
```suggestion
	clusterv1 "sigs.k8s.io/cluster-api-provider-nested/apis/controlplane/v1alpha4"
```

Reply:
will change to `clusterv1`